<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentManageDialog</class>
 <widget class="QDialog" name="StudentManageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
  </property>
  <widget class="QLabel" name="label_sid">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>00000000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>460</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Desgined by VICKEY</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>460</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>http://vickey.me</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_logout">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>LOGOUT</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>500</width>
     <height>370</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::West</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab_tips">
    <attribute name="title">
     <string>Tips</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_info">
    <attribute name="title">
     <string>Information</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>415</width>
       <height>345</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(58, 64, 255);font: 16pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="title">
      <string>Information</string>
     </property>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>281</width>
        <height>191</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>61</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">COLOR:rgb(20, 20, 20);
font: 11pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>name：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_name">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>180</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">COLOR:rgb(20, 20, 20);
font: 11pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Please enter your name</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>41</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">COLOR:rgb(20, 20, 20);
font: 11pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>sex：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_man">
          <property name="styleSheet">
           <string notr="true">COLOR:rgb(20, 20, 20);
font: 11pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>Man</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_female">
          <property name="styleSheet">
           <string notr="true">COLOR:rgb(20, 20, 20);
font: 11pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>Female</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">COLOR:rgb(20, 20, 20);
font: 11pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>E-mail：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_email">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>130</y>
         <width>180</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">COLOR:rgb(20, 20, 20);
font: 11pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Please enter E-mail</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>100</y>
         <width>181</width>
         <height>25</height>
        </rect>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">COLOR:rgb(20, 20, 20);
font: 11pt &quot;微软雅黑&quot;;</string>
       </property>
       <item>
        <property name="text">
         <string>Information &amp; Mathematics</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Petroleum Institute</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Life Science</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">COLOR:rgb(20, 20, 20);
font: 11pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>college：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_tel">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>160</y>
         <width>180</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">COLOR:rgb(20, 20, 20);
font: 11pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Please enter tel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>160</y>
         <width>51</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">COLOR:rgb(20, 20, 20);
font: 11pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>Tel：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>61</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">COLOR:rgb(20, 20, 20);
font: 11pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>sid：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_id">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>180</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">COLOR:rgb(20, 20, 20);
font: 11pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Please enter your  id</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_save">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>280</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">COLOR:rgb(20, 20, 20);font: 11pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_edit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>280</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">COLOR:rgb(20, 20, 20);font: 11pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_score">
    <attribute name="title">
     <string>Query Score</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>445</width>
       <height>345</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 106, 80);font: 16pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="title">
      <string>Score</string>
     </property>
     <widget class="QTableView" name="scoreview">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>425</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>75</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>75</number>
      </attribute>
     </widget>
     <widget class="QTableView" name="scoreview_vip">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>425</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>75</number>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(227, 0, 0)</string>
   </property>
   <property name="text">
    <string>VIP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_head">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
